src/test/chain-simulator/accounts.cs-e2e.ts

```diff
@@ -1187,4 +1187,270 @@ describe('Accounts e2e tests with chain simulator', () => {
+      const miniBlockHashes = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?size=1`);
+      const miniBlockHash = miniBlockHashes.data[0].miniBlockHash;
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?miniBlockHash=${miniBlockHash}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.miniBlockHash).toBe(miniBlockHash);
+      }
+    });
+
+    it('should return transfers with hashes parameter', async () => {
+      const hashes = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?size=2`);
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?hashes=${hashes.data[0].txHash},${hashes.data[1].txHash}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(2);
+      expect(response.data).toContainEqual(hashes.data[0]);
+      expect(response.data).toContainEqual(hashes.data[1]);
+    });
+
+    it('should return transfers with status parameter', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?status=success`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.status).toBe('success');
+      }
+    });
+
+    it('should return transfers with round parameter', async () => {
+      const rounds = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?size=1`);
+      const round = rounds.data[0].round;
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?round=${round}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.round).toBe(round);
+      }
+    });
+
+    it('should return transfers with senderOrReceiver parameter', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?senderOrReceiver=${config.aliceAddress}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.receiver).toBe('erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u');
+      }
+    });
+
+    it('should return transfers with withLogs parameter', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?withLogs=true`);
+      expect(response.status).toBe(200);
+      const hasLogs = response.data.some((transfer: any) => transfer.logs);
+      expect(hasLogs).toBe(true);
```

Comment:
i would check also at least one transfer to have specific (expected) logs

Reply:
done